src/components/Input/input.scss

```diff
@@ -7,9 +7,11 @@
   height: 4em;
   max-width: 100%;
   width: 60em;
+  padding: 1rem;
```

Comment:
Try to group properties by the same class, or if you are doing it alphabetically please be consistent.